docs-src/stargate-develop/modules/develop/examples/result/rest_curl_get_all.result

```diff
@@ -13,21 +13,52 @@
     }, 
     {
       "firstname": "Janesha",
-      "evaluations": 
+      "evaluations": [
         {
-          "2019" : "okay",
-          "2020" : "good"
+          "key": 2019,
+          "value": "okay"
         },
+        {
+          "key": 2020,
+          "value": "good"
+        }
+      ],
       "top_three_tv_shows": [
         "The Magicians",
         "The Librarians",
         "Agents of SHIELD"
       ],
       "favorite color": "grey",
       "current_country": [
-        "France", 
-        "2023-05-22T10:21:00.000Z",
-        "2023-05-23T10:21:00.000Z"
+        "France",
```

Comment:
Ah. Sorry, we are not reverting Date representation changes; it will use the compact notation.